epoch/api/README.md

```diff
@@ -31,6 +31,15 @@ The epoch node exposes the following APIs:
     * It is defined in the rest of this document;
     * It is meant to be exposed on the Internet;
     * Its TCP port is configurable.
```

Comment:
Also, the listen IP address is configurable (default: soon 127.0.0.1, but currently 0.0.0.0, meaning that the node listens on all available IP interfaces.)